CalibPPS/ESProducers/plugins/TotemDAQMappingESSourceXML.cc

```diff
@@ -276,7 +276,9 @@ TotemDAQMappingESSourceXML::TotemDAQMappingESSourceXML(const edm::ParameterSet &
     : verbosity(conf.getUntrackedParameter<unsigned int>("verbosity", 0)),
       subSystemName(conf.getUntrackedParameter<string>("subSystem")),
       sampicSubDetId(conf.getParameter<unsigned int>("sampicSubDetId")),
-      packedPayload(conf.getUntrackedParameter<bool>("multipleChannelsPerPayload", false)),
+      packedPayload(conf.existsAs<bool>("multipleChannelsPerPayload", true)
+                        ? conf.getParameter<bool>("multipleChannelsPerPayload")
+                        : false),
```

Comment:
The `existsAs()` call could be avoided with implementing `fillDescriptions()`  for this module (https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp)